### ISSUE TITLE
Question: FDH with nzbtosickbeard & nzbget

### ISSUE THREAD

**Issue by TomNugter** [clinton-hall/nzbToMedia#825]
Hi,

I'm hope somebody can help me. I'm trying to get Nzbtomedia handle my failed downloads, but i'm having a hard time to get it to work. What i'm trying to archive is that NZBGet gives Sickrage a warning when a download failes on what ever reason so that Sickrage can change it's status from ' snatched'  back to 'wanted' and update 'failed downloads'. 

I'[ve got al the scripts set up, but i'm struggeling with finding the right settings for my setup. I keep getting error's like this:

nzbToSickBeard: [18:31:53] [POSTPROCESS]::SICKBEARD: Unable to figure out what folder to process. If your downloader and SickRage aren't on the same PC make sure you fill out your TV download dir in the config.

Followed by:

nzbToSickBeard: [18:31:53] [ERROR]::MAIN: A problem was reported in the /i-data/2d57e72c/HD/download/scripts/nzbToSickBeard.py script.
nzbToSickBeard: SickBeard: Failed to post-process - Returned log from SickBeard was not as expected.!
Post-process-script nzbToSickBeard.py for The.Brink.S01E01.HDTV.x264-KILLERS failed

I do have Sickrage and NZBget installed on different systems and I think that this is causing my issue. Can somebody please help me finding the right setting?

My nas downloads to /i-data/2d57e72c/HD/download/sickrage when downloading a show pulled from sickrage. I've mounted this on my Pi to /mnt/nas/download/sickrage. /mnt/nas/download/sickrage is also the path I use in sickrage on my Pi.

In the NZBTOSICKRAGE settings i've made the following settings:
sbwatch_dir: /i-data/2d57e72c/HD/download/sickrage
sbremote_path: Enabled (1)
mountpoints: mountPoints=/i-data/2d57e72c/HD/download/sickrage,/mnt/nas/download/sickrage

Are these settings correct? If so, is there anything else I should be looking into?

Any help is much appreciated! If you need anything to support like log or config files, please let me know what you need.

**Comment by TomNugter** [comment]
Just realized that copy/past the settings from autoProcessMedia.cfg might be more use full. The category name I use is 'Series' 

 [[Series]]
        host = 192.168.1.71
        port = 8081
        username = test
        password = test
        ssl = 0
        web_root = ""
        watch_dir = /i-data/2d57e72c/HD/download/sickrage
        fork = sickrage
        delete_failed = 0
        remote_path = 1
        process_method = move
        enabled = 1

[Network]
    # Enter Mount points as LocalPath,RemotePath and separate each pair with '|'
    # e.g. MountPoints = /volume1/Public/,E:\|/volume2/share/,\\NAS\
    mount_points = "/i-data/2d57e72c/HD/download/sickrage,/mnt/nas/download/sickrage"

**Comment by TomNugter** [comment]
I'm still stuck here, some help will be much appreciated. 
I google a lot, but could not find a similar situation. Fiddled around with settings a lot, assuming I mixed up, for examle, localpath and remote path, but nothing.

Maybe the following logs will be give somebody an idea what's failing? Please note that the download was successfull, but the files remain in my TV download folder. (unless I turn on ' scan and progress'  in sickrage, what I've understand is not recommended when using nzbtomedia)

2015-09-19 03:43:19 INFO    ::MAIN: #########################################################
2015-09-19 03:43:19 INFO    ::MAIN: ## ..::[nzbToMedia.pyc]::.. ##
2015-09-19 03:43:19 INFO    ::MAIN: #########################################################
2015-09-19 03:43:19 INFO    ::MAIN: Script triggered from NZBGet Version 15.0.
2015-09-19 03:43:19 INFO    ::MAIN: Auto-detected SECTION:SickBeard
2015-09-19 03:43:19 INFO    ::MAIN: Calling SickBeard:Series to post-process:Z.Nation.S02E02.720p.HDTV.x264-KILLERS
2015-09-19 03:43:21 INFO    ::MAIN: SickBeard:Series fork set to sickrage
2015-09-19 03:43:21 INFO    ::MAIN: FLATTEN: Flattening directory: /i-data/2d57e72c/HD/download/sickrage/Z.Nation.S02E02.720p.HDTV.x264-KILLERS
2015-09-19 03:44:57 POSTPROCESS::SICKBEARD: SUCCESS: The download succeeded, sending a post-process request
2015-09-19 03:44:58 POSTPROCESS::SICKBEARD: Processing folder /mnt/nas/download/sickrage/Z.Nation.S02E02.720p.HDTV.x264-KILLERS
2015-09-19 03:44:58 POSTPROCESS::SICKBEARD: TV_DOWNLOAD_DIR: /mnt/nas/download/sickrage
2015-09-19 03:44:58 POSTPROCESS::SICKBEARD: Unable to figure out what folder to process. If your downloader and SickRage aren't on the same PC make sure you fill out your TV download dir in the config.
2015-09-19 03:44:58 ERROR   ::MAIN: A problem was reported in the /i-data/2d57e72c/HD/download/scripts/nzbToSickBeard.py script

P.s. Please note that i'm fairly new with working with Linux based systems.  It could be possible that I've made some rooky-mistakes somewhere as well.

**Comment by TomNugter** [comment]
It's looks like nobody is able to help me on this. I the issue I describe to common, or do I lack too deliver any information?

**Comment by clinton-hall** [comment]
Or I just have not had any time to get online...

try
```
mount_points = /i-data/2d57e72c/HD/download/sickrage,/mnt/nas/download/sickrage
```
i.e without the `"`
But it looks like this issue is most likely with permissions of the downloaded files. SickRage seems to be unable to look into the directory /mnt/nas/download/sickrage/Z.Nation.S02E02.720p.HDTV.x264-KILLERS

If you ask SickRage to manually postprocess this directory you should see a similar error?

Perhaps you need to play with the NZBGet umask?

**Comment by TomNugter** [comment]
Sorry for being impatient, i was just worried that I just didn't provide enough information. 

I tried filling in mounts point without "s directly in the .cfg file, but after they are replaced. I usually use the UI, there there are no " at all. 

I did a manual postprocess on the files that failed and now they did show up in Sickbeards failed download, so that's something. (the files where already moved). At least it means that my paths are filled correctly.

I'm going look into the umask. This is currently set at UMask=000. Should this be different? (i'm not really experience any other issues with file permissions)

Thanks for the tip anyway, i've got something to look into again and I can cross on cause of the list.
